docs/how_to/menus.rst

```diff
@@ -231,7 +236,22 @@ of :class:`~menus.base.NavigationNode` instances.
   the case ``post_cut`` is ``True``.
 
 ``breadcrumb``
-  Is this breadcrumb call rather than a menu call?
+  Is this a breadcrumb call rather than a menu call?
+
+
+Note: Navigation modifiers can quickly become a performance bottleneck. Each modifier is called
```

Comment:
Please remove *Note:* and add a new heading for this section:
```
Performance issues in menu modifiers
====================================
```

Finally, please move the new *Performance issues in menu modifiers* section after the example of the modifier that marks all node levels.